src/main/resources/wallet.sql

```diff
@@ -0,0 +1,15 @@
+-- Table structure for table 'account'
+-- Use the default Engine INNODB 
+CREATE TABLE IF NOT EXISTS account (
+	id INT NOT NULL AUTO_INCREMENT COMMENT 'unique ID for each account',
+	account_name varchar(20) NOT NULL COMMENT 'the account name' ,
+	fullname varchar(255) NOT NULL COMMENT 'fullname of the account owner',
+	PRIMARY KEY (id),
+	INDEX (id)
+);
+
+-- Some test data
```

Comment:
This should not be part of the DDL. The DDL defines the structure of your tables etc, but initialising data is not part of the DDL, but rather the DML.